lua/nvim-treesitter/parsers.lua

```diff
@@ -603,6 +603,15 @@ list.gowork = {
   maintainers = { "@omertuc" },
 }
 
+list.groovy = {
+  install_info = {
+    url = "https://github.com/Decodetalkers/tree-sitter-groovy",
+    files = { "src/parser.c" },
+    branch = "gh-pages",
```

Comment:
Can't this be pushed to your default branch instead?

Reply:
It is the same as what tree-sitter-sql do, this can make commit clean